package.json

```diff
@@ -4,34 +4,33 @@
 	"description": "Typescript Community Bot",
 	"main": "dist/index.js",
 	"dependencies": {
-		"@typescript/twoslash": "^2.1.0",
-		"algoliasearch": "^4.8.6",
-		"cookiecord": "^0.8.18",
+		"@typescript/twoslash": "^3.2.1",
+		"algoliasearch": "^4.14.2",
+		"discord.js": "^14.6.0",
 		"dotenv-safe": "^8.2.0",
-		"html-entities": "^2.3.2",
+		"html-entities": "^2.3.3",
 		"lz-string": "^1.4.4",
-		"node-fetch": "^2.6.7",
-		"npm-registry-fetch": "^9.0.0",
-		"parse-duration": "^0.4.4",
-		"pg": "^8.3.0",
-		"prettier": "^2.2.1",
-		"pretty-ms": "^7.0.0",
-		"tar": "^6.1.0",
-		"typeorm": "^0.2.25"
+		"npm-registry-fetch": "^14.0.2",
+		"parse-duration": "^1.0.2",
+		"pg": "^8.8.0",
+		"prettier": "^2.7.1",
+		"pretty-ms": "^8.0.0",
+		"tar": "^6.1.12",
+		"typeorm": "^0.3.10",
+		"undici": "^5.12.0"
 	},
 	"devDependencies": {
-		"@types/dotenv-safe": "^8.1.0",
+		"@types/dotenv-safe": "^8.1.2",
 		"@types/lz-string": "^1.3.34",
-		"@types/node": "^13.7.0",
-		"@types/node-fetch": "^2.5.8",
-		"@types/npm-registry-fetch": "^8.0.0",
-		"@types/prettier": "^2.2.3",
-		"@types/tar": "^4.0.4",
-		"@types/ws": "^7.2.1",
-		"husky": "^4.2.5",
-		"pretty-quick": "^2.0.1",
-		"ts-node-dev": "^1.0.0-pre.60",
-		"typescript": "^4.1.3"
+		"@types/node": "^16",
+		"@types/npm-registry-fetch": "^8.0.4",
+		"@types/prettier": "^2.7.1",
+		"@types/tar": "^6.1.3",
+		"@types/ws": "^8.5.3",
+		"husky": "^8.0.2",
+		"pretty-quick": "^3.1.3",
+		"ts-node-dev": "^2.0.0",
+		"typescript": "^4.9.3"
 	},
 	"husky": {
```

Comment:
For current version of husky I believe this should be removed, in favor of running:

```
npx husky install
npx husky add .husky/pre-commit "pretty-quick --staged"
```
And adding `"prepare": "husky install"` as a package script